### PR TITLE
Add returning of network packets after insert

### PR DESCRIPTION
**Pull request by net-illia-stetsenko**

### CODE REVIEW

File: net-gateway/src/endpoints/files/pcap_files.rs [hunk 1]
```diff
@@ -67,13 +72,54 @@ async fn pcap_files(
...
+                    }));
+                }
+                let response = response.unwrap();
+                if response.get_envelope_type() != NetworkPacketDTO::get_data_type() {
+                    return HttpResponse::InternalServerError().json(serde_json::json!({
+                        "error": "Invernal Server Error",
+                        "message": "Received wrong data type after decoding, double check the data you want to decode"
+                    }));
+                }
+                packets.push(NetworkPacketDTO::decode(response.get_data()).into());
```
Comment (net-yehor-tretiakov) [inline net-gateway/src/endpoints/files/pcap_files.rs:106]: Dont you wanna make this one work in parallel?

Comment (net-yehor-tretiakov) [reply net-gateway/src/endpoints/files/pcap_files.rs:106]: Like, I mean...
Youll prob understand me

**Review by net-yehor-tretiakov — COMMENTED** [review]
Take a look at the comment

File: net-gateway/src/endpoints/files/pcap_files.rs (hunk 1 above)
Comment (net-illia-stetsenko) [reply net-gateway/src/endpoints/files/pcap_files.rs:106]: Do you mean decoding packets?

**Review by net-yehor-tretiakov — APPROVED** [review]

**Pull request merged**